src/common/transformations/include/transformations/common_optimizations/concat_to_broadcast.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API ConcatToBroadcast;
+
+}  // namespace pass
+}  // namespace ov
+
+/**
+ * TBA
+ */
+
+class ov::pass::ConcatToBroadcast : public ov::pass::MatcherPass {
```

Comment:
please add unit tests to cover this Transformation

Reply:
Added.